assets/all.settings.php

```diff
@@ -58,12 +54,6 @@
     CURLOPT_PROXY_SSL_VERIFYHOST => 0,
     CURLOPT_PROXY_SSL_VERIFYPEER => FALSE,
   ];
-  // Specify non-HTTP versions of endpoints. This is required to make Cypress
-  // mocking work. It does not support ignoring self-signed certificates from
-  // Wiremock.
-  // Service base urls for the external APIs.
-  $config['dpl_fbs.settings'] = ['base_url' => 'http://fbs-openplatform.dbc.dk'];
-  $config['dpl_publizon.settings'] = ['base_url' => 'http://pubhub-openplatform.test.dbc.dk'];
```

Comment:
They are https in the configuration files in config sync

Reply:
Since you removed the swapping of urls from all.settings.php we need to have another way of swapping.
We need to maintain that looks at the[ env: "ci"](https://github.com/danskernesdigitalebibliotek/dpl-cms/blob/76a9d5c3d9f2cc1b9fec24c6b887f363dbe281e8/assets/all.settings.php#L49) and swaps ur to use http.
Does it make sense?

Reply:
Fixed :-D